UPGRADE-3.4.md

```diff
@@ -411,6 +411,53 @@ Validator
  * Not setting the `strict` option of the `Choice` constraint to `true` is
    deprecated and will throw an exception in Symfony 4.0.
 
+ * The `strict` option of the `Email` constraint is deprecated and will
+   be removed in Symfony 4.0, use the `mode` option instead
+
+   Before:
+
+   ```php
+   use Symfony\Component\Validator\Constraints as Assert;
+
+   /**
+    * @Assert\Email(strict=true)
+    */
+   private $property;
+   ```
+
+   After:
+
+   ```php
+   use Symfony\Component\Validator\Constraints as Assert;
+
+   /**
+    * @Assert\Email(mode="strict")
+    */
+   private $property;
+   ```
+
+ * The constructing the `EmailValidator` with a boolean is deprecated and will
```

Comment:
Calling the `EmailValidator` with a boolean constructor argument is deprecated [...]